platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -135,9 +137,17 @@ BundleContext getBundleContext() {
         List<MigrationWarning> warnings = new LinkedList<>();
 
         for (ServiceReference<Migratable> serviceReference : getMigratableServiceReferences()) {
-
             Migratable migratable = getBundleContext().getService(serviceReference);
+
+            LOGGER.debug("Exporting {}", migratable.getDescription());
+
+            Stopwatch stopwatch = Stopwatch.createStarted();
```

Comment:
I didn't want all the `Migratables` to have to add their own timing code so I moved it outside. We could consider timing only the data related `Migratables` in the future when we make a distinction between configuration and data `Migratables`.